README.md

```diff
@@ -2,7 +2,64 @@
 
 ## About
 
-This repository defines how to store geospatial information (point, lines, polygons) in Parquet.
+This repository defines how to store geospatial information (point, lines, polygons) in [Parquet](https://parquet.apache.org/), a popular 
+[columnar storage format](https://en.wikipedia.org/wiki/Column-oriented_DBMS). Our goal is to standardize how geospatial data is represented
+in Parquet to further geospatial interoperability among tools using Parquet today, and hopefully help push forward what's possible with 
+'cloud-native geospatial' workflows. 
 
 **Warning:** This is a completely unreleased specification, with several outstanding updates to make. If you are excited about the potential 
-please sound in on the issues and contribute PR's!
+please sound in on the issues and contribute PR's! But please do not start converting all your data to this format, as it will change 
+(and possibly in a backwards incompatible way, see the [versioning](#versioning) section below.
+
+Early contributors include developers from Voltron Data, Microsoft, Carto, Azavea, Planet & Unfolded. Anyone is welcome to join us, by building 
```

Comment:
Do we also want to list open source communities here in addition to companies?

Reply:
Sure! That'd be geopandas, right? And geotrellis? Any others?

Reply:
Could add OpenLayers too, with @tschaub.

Reply:
In case we want to add communities :+1: GT is here! Could be a broader one (LocationTech? GeoTrellis + GeoWave + GeoMesa + JTS + others)

Reply:
[Vis.gl](https://github.com/visgl) 🙂

Reply:
Added! I was trying to figure out how to represent your open source communities @kylebarron - vis.gl is perfect.